logstash-core/build.gradle

```diff
@@ -181,8 +181,6 @@ dependencies {
     implementation 'org.codehaus.janino:janino:3.1.0'
     implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:${jacksonVersion}"
     implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${jacksonVersion}"
```

Comment:
Thanks for `jrjackson` upgrade. Now this solves [snakeyaml 1.33 CVEs](https://github.com/FasterXML/jackson-dataformats-text/pull/390).